[FEATURE] #99834 - Add two PSR-14 events around auto-create redirects (#2772)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/Events/Events/Redirects/AfterAutoCreateRedirectHasBeenPersistedEvent.rst

..  include:: /Includes.rst.txt
..  index:: Events; AfterAutoCreateRedirectHasBeenPersistedEvent
..  _AfterAutoCreateRedirectHasBeenPersistedEvent:

============================================
AfterAutoCreateRedirectHasBeenPersistedEvent
============================================

..  versionadded:: 12.3

The PSR-14 event
:php:`\TYPO3\CMS\Redirects\Event\AfterAutoCreateRedirectHasBeenPersistedEvent`

> allows extension authors to react on persisted auto-created redirects. This

Reviewer: ```suggestion
allows extensions to react on persisted auto-created redirects. This
```

Author: How can a thing react?

Reviewer: How can the extension author react when he is not there?

Author: The developer reacts on an event programmatically...

Reviewer: I would have said, the developer instructs the extension to react. But whatever. Keep it like this if you want

> allows extension authors to react on persisted auto-created redirects. This

Reviewer: ```suggestion
allows extension authors to react on persisted auto-created redirects. This event
```

can be used to call external APIs or perform other tasks based on the real
persisted redirects.

..  note::

>     To handle later updates or react on manual created redirects in the backend

Reviewer: ```suggestion
    To handle later updates or react on manually created redirects in the backend
```
